Libraries/NativeAnimation/RCTNativeAnimatedModule.m

```diff
@@ -40,10 +42,18 @@ - (void)setBridge:(RCTBridge *)bridge
 
   _animationNodes = [NSMutableDictionary new];
   _animationDrivers = [NSMutableDictionary new];
+  _eventAnimationDrivers = [NSMutableDictionary new];
   _activeAnimations = [NSMutableSet new];
   _finishedAnimations = [NSMutableSet new];
   _updatedValueNodes = [NSMutableSet new];
   _propAnimationNodes = [NSMutableSet new];
+
+  [bridge.eventDispatcher addDispatchObserver:self];
+}
+
+- (void)dealloc
+{
+  [self.bridge.eventDispatcher removeDispatchObserver:self];
```

Comment:
The `bridge` property is defined in the parent class so I can't use the ivar directly in this case.